usaspending_api/awards/models.py

```diff
@@ -455,7 +455,7 @@ class TransactionAssistance(DataSourceTrackedModel):
     uri = models.TextField(blank=True, null=True)
     cfda_number = models.TextField(blank=True, null=True, verbose_name="CFDA Number")
```

Comment:
It won't require any extra updates, but front end may need to make some changes if we remove those fields. I'm totally game to remove them though. @mab6bf any thoughts on that?